dialogy/plugins/text/duckling_plugin/__init__.py

```diff
@@ -597,11 +597,12 @@ def __create_req_body(
         :rtype: Dict[str, Any]
         """
         dimensions = self.dimensions
-        self.activate_latent_entities = use_latent or self.activate_latent_entities
+        activate_latent_entities = use_latent or self.activate_latent_entities
```

Comment:
Can we have a test case to ensure the issue #124 is addressed by this change?
It will also prevent accidentally breaking this in a future release during refactor exercises.

Reply:
skipping test case for this change since use_latent is reliable.